C_samples/main.c

```diff
@@ -0,0 +1,1078 @@
+    printf("----------------------------------------------------------------------\r\n");
+   
+    printf("Proceeding with testing functions with one table in and no out parameters...\r\n");
+    test_one_table_in_no_out_params_functions();
+    printf("Testing functions with one table in and no out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with no in and one table out parameters...\r\n");
+    test_no_in_one_table_out_params_functions();
+    printf("Testing functions with no in and one table out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with one table in and one out parameters...\r\n");
+    test_one_table_in_one_out_params_functions();
+    printf("Testing functions with one table in and one out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with one in and one table out parameters...\r\n");
+    test_one_in_one_table_out_params_functions();
+    printf("Testing functions with one in and one table out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with one table in and one table out parameters...\r\n");
+    test_one_table_in_one_table_out_params_functions();
+    printf("Testing functions with one table in and one table out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with two tables in and one table out parameters...\r\n");
+    test_two_tables_in_one_table_out_params_functions();
+    printf("Testing functions with two tables in and one table out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with one table in and two tables out parameters...\r\n");
+    test_one_table_in_two_tables_out_params_functions();
+    printf("Testing functions with one table in and two tables out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Proceeding with testing functions with two tables in and two tables out parameters...\r\n");
+    test_two_tables_in_two_tables_out_params_functions();
+    printf("Testing functions with two tables in and two tables out parameters ended succesfully.\r\n");
+    
+    printf("----------------------------------------------------------------------\r\n");
+
+    printf("Testing finished, all tests passed.\r\n");
+}
```

Comment:
`main` is defined with `int` return type, hence it should return something

Reply:
Resolved.